Add German Readme (#8712)
TeamNewPipe/NewPipe: doc/README.de.md

Bei Methode 5 wird für jede APK ein zufälliger Signierschlüssel von GitHub Actions generiert, d.h. Updates sind unmöglich.
Bei jeder neuen APK Installation muss ein Backup gemacht werden und die Daten wiederhergestellt werden.

Falls du aus welchem Grund auch immer die Quelle (Methode) ändern willst (z.B. weil irgendetwas in NewPipe nicht mehr funktioniert und F-Droid nicht das neuste Update mit dem Fix hat), empfehlen wir folgende Herangehensweise:
1. Mache ein Back-up über _Einstellungen > Inhalt > Datenbank exportieren_ damit dein Verlauf, deine Abos und Playlists nicht verloren gehen
2. Deinstalliere NewPipe
3. Lade die APK von der neuen Quelle herunter und installiere es
4. Importiere die Daten aus Schritt 1 über _Einstellungen > Inhalt > Datenbank importieren_

<!-- What is an unsupported app anyway? -->
<b>Hinweis: Wenn du eine Datenbank in die offizielle App importierst, stelle sicher, dass sie _von_ der offiziellen App exportiert wurde.
Wenn du eine Datenbank von einer nicht offiziellen APK exportierst, werden Dinge vorraussichtlich kaputt gehen.
So eine Aktion wird nicht unterstützt und du solltest es nur in Erwägung ziehen, wenn du weißt, was du tust.</b>

## Beitrag

> Egal ob du neue Ideen, Übersetzungen, Designvorschläge, kleine Code-Bereinigungen, oder sogar große Code-Verbesserungen hast, jegliche Unterstützung ist immer gern gesehen.
> Die App wird mit _jedem_ Beitrag besser und besser - egal wie groß oder klein er sein mag!
> Wenn du dich einbringen willst, sehe dir die [Beitragshinweise](.github/CONTRIBUTING.md) an.

Reviewer: I rewrote this to avoid double _groß_

```suggestion
Egal ob du neue Ideen, Übersetzungen, Designvorschläge, kleine Code-Bereinigungen, oder sogar große Code-Verbesserungen hast, jegliche Unterstützung ist immer gern gesehen.
Die App wird mit _jedem_ Beitrag besser und besser - egal wie viel Arbeit in ihn gesteckt wird!
Wenn du dich einbringen willst, sehe dir die [Beitragshinweise](.github/CONTRIBUTING.md) an.
```

<a href="https://hosted.weblate.org/engage/newpipe/de/">
<img src="https://hosted.weblate.org/widgets/newpipe/de/287x66-grey.png" alt="Übersetzt" />
</a>

## Spenden
Wenn dir NewPipe gefällt, kannst du uns gerne durch eine Spende unterstützen.
Wir bevorzugen Liberapay, da es sowohl quelloffen als auch nicht gewinnorientiert ist.

> Für weitere Informationen über Spenden an NewPipe, gehe auf unsere [Website](https://newpipe.net/donate).

Reviewer: I think thats the more common term

```suggestion
Für weitere Informationen über Spenden an NewPipe, besuche unsere [Website](https://newpipe.net/donate).
```

    <td><samp>16A9J59ahMRqkLSZjhYj33n9j3fMztFxnh</samp></td>
  </tr>
  <tr>
    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/22/Bountysource.png/320px-Bountysource.png" alt="Bountysource" width="190px"></a></td>
    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="../assets/bountysource_qr_code.png" alt="Gehe zu NewPipe auf bountysource.com" width="100px"></a></td>
    <td><a href="https://www.bountysource.com/teams/newpipe/issues"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f" height="30px" alt="Sieh nach, wie viel Bounties du bekommen kannst."></a></td>
  </tr>
</table>

## Privacy Policy

Das NewPipe Projekt hat zum Ziel, eine private und anonyme Benutzung von Web-basierten Medien-Dienste zu ermöglichen.
Daher sammelt die App keinerlei Daten ohne deine Zustimmung.
NewPipe's Datenschutzbestimmungen erklären im Detail, welche Daten gesendet und gespeichert werden, wenn du einen _Crash Report_ einreichst, oder ein Kommentar auf unserem Blog hinterlässt.
Du findest das Dokument [hier](https://newpipe.net/legal/privacy/).

## Lizenz

> [![GNU GPLv3 Image](https://www.gnu.org/graphics/gplv3-127x51.png)](https://www.gnu.org/licenses/gpl-3.0.en.html)  

Reviewer: ```suggestion
[![GNU GPLv3 Image](https://www.gnu.org/graphics/gplv3-127x51.png)](https://www.gnu.org/licenses/gpl-3.0.de.html)  
```

> NewPipe ist freie Software: Du kannst es nach Beliebeb benutzen, studieren, teilen, und verbessern.

Reviewer: ```suggestion
NewPipe ist freie Software: Du kannst es nach Belieben benutzen, studieren, teilen, und verbessern.
```

> Du kannst es insbesondere unter den Bedingungen der [GNU General Public License](https://www.gnu.org/licenses/gpl.html), wie von der Free Software Foundation veröffentlicht, entweder Version 3 der Lizenz oder (nach deiner Wahl) jede spätere Version, weiter verbreiten und/oder verändern.

Reviewer: ```suggestion
Du kannst es insbesondere unter den Bedingungen der [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.de.html), wie von der Free Software Foundation veröffentlicht, entweder Version 3 der Lizenz oder (nach deiner Wahl) jede spätere Version, weiter verbreiten und/oder verändern.
```
